docs/pages/buttons.vue

```diff
@@ -51,11 +51,13 @@
           hyperlink-like appearance for deemphasized actions, or actions inline within text
         </li>
       </ul>
-      <!-- TODO: add a flat secondary icon button to these examples below. See Google doc for example -->
       <DocsShow>
-        <KButton text="Raised button" :primary="true" appearance="raised-button" />
-        <KButton text="Flat button" :primary="true" appearance="flat-button" />
-        <KButton text="Basic link" :primary="true" appearance="basic-link" />
+        <KButtonGroup>
+          <KButton text="Raised button" :primary="true" appearance="raised-button" />
+          <KButton text="Flat button" :primary="true" appearance="flat-button" />
+          <KIconButton tooltip="A tooltip" text="Flat button" :primary="false" icon="plus" appearance="flat-button" />
```

Comment:
@jtamiace @nucleogenesis realizing that 'primary' might not be implemented for `KIconButton`?

![image](https://user-images.githubusercontent.com/2367265/88116510-8f0ba400-cb6d-11ea-814d-aa138c0dd324.png)

Reply:
it might be that in fact we don't want to create a primary state for icon buttons. I think that would be totally reasonable.

Reply:
Yeah at the moment we don't currently use raised or primary icon buttons anywhere across all the apps

Reply:
At least, not ones that we'd want to standardize